### ISSUE TITLE
[DX] change the names of created and changed columns in the node table

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#3049]
## Describe your issue or idea

We could change the names of the `created` and `changed` columns in the node table to `created_date` and `updated_date`. 

The past-tense verb column names present some ambiguity as to whether the column contains a boolean or a value, in this case, a date. If we include the word `_date` it will be clear to people at first glance what data is contained within.

**Comment by serundeputy** [comment]
I'm thinking `2.x` for sure.
This could and probably will affect contrib and custom modules.

**Comment by BWPanda** [comment]
I'm all for the change, but I agree with @serundeputy

**Comment by oadaeh** [comment]
This seems unnecessary to me. It will make it so that people moving practically anything from Drupal to Backdrop, including things like views, will have more work to do to make it happen.
Also, those timestamps include a time, not just a date, which a name like `changed_date` does not imply.

**Comment by herbdool** [comment]
I agree. On balance this provides little benefit for the amount of work it places on all developers.

**Comment by bugfolder** [comment]
> I'm hesitant to change the column names in the 1.x cycle, as there could be any number of custom or contrib projects that are currently querying the node table directly. 

LIke `uc_new_expire`, `webform`, and several custom modules on one of my sites (based on a search for `.created` and then examining to see if it was in a `node` table query).

**Comment by avpaderno** [comment]
While *created* could be taken as a Boolean value, the description given in [`node_schema()`](https://docs.backdropcms.org/api/backdrop/core%21modules%21node%21node.install/function/node_schema/1) should make clear what value that field contains.

> The Unix timestamp when the node was created.

If it was just a matter of renaming a database row, I would be in with the change.  
I do not see any pro to force modules to change their code, though.

It is not also a matter of changing two database rows, though. For nodes, there is also the *scheduled* row, but there are database tables with similar rows that should be changed.
